api/src/test/java/org/openmrs/module/registrationcore/api/RegistrationCoreServiceTest.java

```diff
@@ -37,25 +44,36 @@
 import org.openmrs.module.registrationcore.api.biometrics.model.BiometricData;
 import org.openmrs.module.registrationcore.api.biometrics.model.BiometricSubject;
 import org.openmrs.module.registrationcore.api.biometrics.model.Fingerprint;
+import org.openmrs.module.registrationcore.api.impl.RegistrationCoreProperties;
+import org.openmrs.module.registrationcore.api.impl.RegistrationCoreServiceImpl;
+import org.openmrs.module.registrationcore.api.mpi.common.MpiPatient;
+import org.openmrs.module.registrationcore.api.mpi.common.MpiProvider;
 import org.openmrs.test.Verifies;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.test.annotation.NotTransactional;
 
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Date;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
+import java.util.GregorianCalendar;
+import java.util.Calendar;
 
 import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
 
 /**
  * Tests {@link RegistrationCoreService} .
  */
 public class RegistrationCoreServiceTest extends RegistrationCoreSensitiveTestBase {
-	
+
+	@InjectMocks
+	private RegistrationCoreService serviceForMocks = new RegistrationCoreServiceImpl();
+	@Mock
+	private MpiProvider mpiProvider;
+	@Mock
```

Comment:
I thought Adding @Mock automatically leads to an instance getting creating and assigned implying you don't need to manually assign the variable a new instance